src/app/custom-editor/custom-editor.component.html

```diff
@@ -20,7 +20,8 @@
           </svg>
           <span class="status-text">{{unsavedChanges ? 'Unsaved' : 'Saved'}}</span>
       </div>
-      <div class="status" [title]="editorFocused ? 'Editor is in focus, commands can be run: press F1 for options' : 'Editor is not in focus, click on the editor to focus'">
+      <div class="status"
+           [title]="editorFocused ? 'Editor is in focus, commands can be run: press F1 for options' : 'Editor is not in focus, click on the editor to focus'">
```

Comment:
This text feels a bit long to read.

Might I suggest "Editor focused, press F1 to run commands" and "Editor unfocused, click on the editor to focus"